Makefile.create_config

```diff
@@ -24,4 +24,5 @@ src/ocamlbuild_config.ml: force
 	echo 'let so = "$(SO)"'; \
 	echo 'let ext_dll = "$(EXT_DLL)"'; \
 	echo 'let exe = "$(EXE)"'; \
+	echo 'let version = "$(shell cat VERSION)"'; \
```

Comment:
This is an one-line OCaml script--I think running it using the ocaml toplevel is perfectly fine.